pkg/deploy-manager/storagecluster.go

```diff
@@ -91,6 +91,9 @@ func (t *DeployManager) DefaultStorageCluster() (*ocsv1.StorageCluster, error) {
 					},
 				},
 			},
+			NFS: &ocsv1.NFSSpec{
+				Enable: true,
+			},
```

Comment:
Does this mean NFS is enabled by default? If so, this should be made disabled by default.

Reply:
> Does this mean NFS is enabled by default? If so, this should be made disabled by default.

This is only being used for e2e and to start the cluster with scripts
https://github.com/red-hat-storage/ocs-operator/blob/df8317888c7f65d59fe1840a3622843fd061efd2/pkg/deploy-manager/storagecluster.go#L28